test/Microsoft.Azure.Devices.Edge.Test.Common/EdgeDevice.cs

```diff
@@ -69,6 +76,8 @@ public class EdgeDevice
         public static async Task<EdgeDevice> GetOrCreateIdentityAsync(
             string deviceId,
```

Comment:
Should we be concerned about maximum length of the deviceId here?

Reply:
According to https://github.com/MicrosoftDocs/azure-docs/blob/master/includes/iot-hub-limits.md the max length is 128. I will add a check in this function to be sure that we stay below that. I could see some tests in the future perhaps overstepping?

Reply:
Doesn't the cloud call to create the device fail if the requested device ID is too long? I'd rather not duplicate that check on the client if it isn't needed.

Reply:
Agreed - Also, if the test writer is using device IDs that are too big, they would notice with failing tests :)